<!DOCTYPE html>
<html lang="en">
<head>
    <link rel="stylesheet" type="text/css" href="/Css/style.css">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>

<body>
    <!--Start-->
    <div class="container">
        <p>
            Dear [name], 
        </p>
        <p>
            Thank you for choosing to assess your Coronavirus Genetic Susceptibility and we hope you have agreed to participate in our survey and research efforts to provide further insight into the role of your genetics in SARS CoV-2 virus response. It is clear from the medical experts that different individuals respond differently to SARS CoV-2 infection with some being asymptomatic while others have severe complications with varying degrees of morbidity and mortality.   Although lifestyle, environmental factors, and pre-existing health conditions may be contributing factors to your risk of SARS-CoV-2 infection and response, your genes may also influence how you respond to exposure to SARS CoV-2 virus.  
        </p>
        <p>
            Breakthrough Genomics has had a long-standing commitment to helping you better understand your genome to empower you on your personal health journey and to help you live a healthy and long life.  As the pandemic started to unfold, our team used our proprietary artificial intelligence and deep machine learning powered database to identify the genes and variants that have been associated with SARS CoV-2 from 2019, SARS-CoV-1 from the SARS outbreak in 2003, as well as the middle east respiratory syndrome-related coronavirus (MERS-CoV) from 2012.  In many cases, these studies had either not been published in peer-reviewed journals, or had not been confirmed by other scientific groups.  As such, these finding are considered “anecdotal” by the regulatory agencies much like some of the drugs that are currently being repurposed to treat COVID-19 patients.  Most of the genes and variants we identified make sense as either impacting the infection or replication of the virus or impacting your immune response to viral infection.  Rather than wait for formal studies, our team felt that this data was so compelling that we should make this information available now on a research or informational basis; especially given the acute and rapid nature of COVID-19 development in some patients and as we transition to more relaxed precautionary measures and consider returning to work, travel, and our normal routine activities.  We hope to that this information may be useful in deciding how you and your family can stay safe and healthy during these challenging times.
        </p>
        <p>
            Your report is separated into 4 major categories, 1) virus infection or replication, 2) innate immunity and antiviral defense 3) inflammatory cytokine responses, or 4) other immune related responses.   Sections 1 and 2 relate to your susceptibility (or resistance) to virus infection and persistence of infection while Sections 3 and 4 are related to your immune response to viral infection so may influence whether or not you may be susceptible to developing SARS as well as the level of severity.  If any of these categories do not appear in your report, then you did not carry any of the variants identified in this category.  
        </p>
        <p>
            Your DNA sequences have been compared to a reference genome which is the standard for genetic analysis.  Any difference from this reference genome is considered a “variant”. Except for the X and Y chromosomes, you have two chromosomes each of which carries the gene sequences associated with the variant.  We identify the sequences of the variant region present on each chromosome (referred to the allele) and list the nucleotides of each as your genotype (listed as XX in Table 1 below).  The variant nucleotide of the allele will be listed in the results text.  The allele frequency is a percentage that represents how often this variant is observed in the general population. In many cases, these variant may be rare constituting < 1% of or of low frequency low-frequency (1–5%) of the general population (listed as N.NN% below).  The gene associated with the variant is listed as a standard abbreviation for the gene (listed as GEN below). Information about the gene, its function, and relation to coronavirus and SARS is presented above variants in the full report section.  The gene variant or polymorphism is presented based on its location in the gene and is listed in two different ways; for those variants that affect the protein sequence, the reference amino acid is listed as a single letter (G) followed by the position (999) then the variant encoded amino acid (A).  For non-coding variants, the location is listed first as the number of nucleotides relative to the coding sequence where a minus (-) indicates upstream and a plus (+ )downstream of the coding sequences which are then followed by the reference nucleotide (T).  For example, -555T corresponds to a T sequence located 555 nucleotides upstream of the start codon.  Finally, we have included a susceptibility gauge with each variant where the green is resistance or protection and Red represents susceptible to virus infection or SARS development.  7 different levels are presented which correspond to highly resistant, resistant, mildly resistant, normal susceptibility, mildly susceptible, susceptible, and highly susceptible.
        </p>

        <!--Firgue 1-->

        <div class="figure">
            <h5>
                Figure 2. Key to Gauges
            </h5>
            <table>
                <tr>
                    <th>Highly Resistant</th>
                    <th>Resistant</th>
                    <th>Mildly Resistant</th>
                    <th>Normal Susceptibility</th>
                    <th>Mild Susceptibility</th>
                    <th>Susceptible</th>
                    <th>Highly Susceptible</th>
                </tr>
                <tr>
                    <td><img width="130px" src="/images/level.png" alt=""></td>
                    <td><img width="130px" src="/images/resistant.png" alt=""></td>
                    <td><img width="130px" src="/images/midly-resistant.png" alt=""></td>
                    <td><img width="130px" src="/images/normal-susceptibility.png" alt=""></td>
                    <td><img width="130px" src="/images/mild-susceptibility.png" alt=""></td>
                    <td><img width="130px" src="/images/susceptible.png" alt=""></td>
                    <td><img width="130px" src="/images/highly-susceptible.png" alt=""></td>
                </tr>
            </table>
        </div>

        <p>The results section of each variant describes the variant, its potential impact on the gene it is associated with and how that may impact virus infection, general immune function, SARS development.  Information regarding protection, resistance, or susceptibility is shown.  The scientific journal or reference material is indicated in [brackets] and presented in detail at the end of the report.</p>
        <p>If you would like more information about your genes, gene variants, and how your genes influence diseases see https://www.genome.gov/About-Genomics/Introduction-to-Genomics.</p>
        <p>Even though the results may suggest you are highly resistant to infection SARS CoV2, it does not guarantee that you cannot be infected.  Likewise, a highly susceptible variant may not necessarily mean you will develop severe SARS if infected.  Many individuals are deciding not to follow the recommended precautionary measures for avoiding COVID-19, you may want to consider the results of your susceptibility assessment as well as lifestyle, environmental factors, and pre-existing health conditions in this regard.</p>
        <p>If you would like to join our mailing list for updates on the clinical utility of these variants in SARS COV-2 and COVD-19, please send us an email at covid@btgenomics.com.</p>
        <p>If you would like to discuss your assessment results, we can refer you to a genetic counselor or physician.  Please contact us at info@btgenomics.com.</p>
    </div>
</body>
</html>